index.js

```diff
@@ -20,56 +19,50 @@ var readFile = function(dataFile, appContext, appEntryPoint) {
   });
 };
 
-var parseAppData = function(dataObj, appContext, appEntryPoint) {
+// Data is expected in the format that is output by using the following query:
+// curl -G 'http://localhost:8086/query' --data-urlencode 'db=raptor' --data-urlencode
+// "q=select percentile(value, 95) from measure where metric='visuallyLoaded' and
+// context='communications.gaiamobile.org@dialer' and time > now() - 7d group by
+// revisionId; select * from annotation"
```

Comment:
Should probably match the annotation query up with the measure query:

`select * from annotation where test = 'coldlaunch' and time > now() - 7d`

Just so we don't return the world. :)